sass/_Variables.scss

```diff
@@ -4,6 +4,19 @@
 $coveo-orange: #f58020;
 $coveo-blue: #0059b3;
 $coveo-red: #dc291e;
+// New corporate main colors
+$color-primary: #1372ec;
+$color-primary-variant: #215cd3;
+$color-on-primary: #ffffff;
+$color-secondary: #333357;
+$color-secondary-variant: #333357;
+$color-on-secondary: #ffffff;
+$color-background: #ffffff;
+$color-background-variant: #f6f7f9;
+$color-on-background: #282829;
+$color-divider: #e5e8e8;
+$color-error: #ce3f00;
+$color-on-error: #ffffff;
```

Comment:
Copied from [Atomic's colors](https://github.com/coveo/ui-kit/blob/master/packages/atomic/tailwind.config.js#L14).

Reply:
You can just copy the primaries, I don't think we'll require the rest, and they're not used anywhere currently